test/functional/api/v1/doorkeeper/profiles_controller_test.rb

```diff
@@ -0,0 +1,55 @@
+require 'test_helper'
+
+class Api::V1::Doorkeeper::ProfilesControllerTest < ActionController::TestCase
+  setup do
+    @user = create(:user)
+    @oauth_token = create(:oauth_access_token, resource_owner_id: @user.id)
+    @request.headers["Authorization"] = "Bearer #{@oauth_token.token}"
+  end
+
+  def response_body
+    JSON.parse @response.body
```

Comment:
Perhaps should cache this or not make it a method call? That's 6 calls in the "include user :id, :handle, :email" test.